src/ape/api/providers.py

```diff
@@ -730,6 +730,14 @@ def estimate_gas_cost(self, txn: TransactionAPI, **kwargs) -> int:
         """
 
         txn_dict = txn.dict()
+
+        # "auto" does not usually work
```

Comment:
```suggestion
```

`auto` only means get into the method we are in now.

I don't see a problem with popping it out at this point! That makes sense.